src/kernel/thread/fixed_stack.h

```diff
@@ -12,7 +12,7 @@
 
 struct thread_stack {
 	void              *stack;         /**< Allocated thread stack buffer. */
-	size_t             stack_sz;      /**< Stack size. */
+	size_t stack_sz;                  /**< Stack size. */
```

Comment:
The former version have members aligned (probably with some extra indentation), but new version is worse